crates/db_views/src/post_view.rs

```diff
@@ -418,8 +429,9 @@ impl<'a> PostQueryBuilder<'a> {
       query = query.filter(post_read::id.is_null());
     }
 
-    if let Some(languages) = self.languages {
-      // TODO
+    // Filter out the rows with missing languages
+    if self.my_local_user_id.is_some() {
+      query = query.filter(local_user_language::id.is_not_null());
```

Comment:
Not sure if this will work for posts with an undetermined language.

Reply:
I would expect that users have to specify explicitly that they want to see posts with undetermined language. Anyway we can fine tune later when testing this.